src/components/organisms/DustStorm/DustStorm.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useCallback, useEffect } from "react";
+import "./DustStorm.scss";
+import { WithId } from "utils/id";
+import { AnyVenue } from "types/Firestore";
+import firebase from "firebase";
+import { useHistory } from "react-router-dom";
+import { OnlineStatsData } from "../../../../src/types/OnlineStatsData";
+
+const getRandomInt = (max: number) => {
+  return Math.floor(Math.random() * Math.floor(max + 1));
+};
+
+interface PoLuckButtonProps {
+  openVenues?: Array<WithId<AnyVenue>>;
+  afterSelect: () => void;
+}
+
+const PotLuckButton: React.FC<PoLuckButtonProps> = ({
+  openVenues,
+  afterSelect,
+}) => {
+  const history = useHistory();
+  const goToRandomVenue = useCallback(() => {
+    const ExperiencesOrArtpieces = openVenues?.filter(
+      (venue) => venue.template === "zoomroom" || venue.template === "artpiece"
+    );
+
+    if (!ExperiencesOrArtpieces) return;
+
+    const randomVenue =
+      ExperiencesOrArtpieces[getRandomInt(ExperiencesOrArtpieces?.length - 1)];
+    afterSelect();
+
+    if (randomVenue?.template === "artpiece")
+      history.push(`/in/${randomVenue.id}`);
+    if (randomVenue?.template === "zoomroom")
```

Comment:
nit: Can we use the `settings.ts` sets of venues so we are covered if we add another template